tutorail/README.md

```diff
@@ -0,0 +1,13 @@
+## CodeQuery使用教程
```

Comment:
You have a typo in the name of the directory, expected `tutorial`, not `tutorail`

Reply:
ok